pkg/providers/aws/cluster_network_provider.go

```diff
@@ -153,12 +155,21 @@ func (n *NetworkProvider) CreateNetwork(ctx context.Context, vpcCidrBlock *net.I
 		//By default, `integreatly.org/clusterID`.
 		//
 		//NOTE - Once a VPC is created we do not want to update it. To avoid changing cidr block
+		clusterVPC, err := getClusterVpc(ctx, n.Client, n.Ec2Api, n.Logger)
+		if err != nil {
+			return nil, errorUtil.Wrap(err, "failed to get cluster vpc")
```

Comment:
hey @laurafitzgerald, just more of a question, I see that the getClusterVpc can return different errors, not entirely sure how does the errorUtil.Wrap work, but from my initial understanding, if this error is met we are going to return "failed to get cluster vpc" - if so, maybe it would be better to return the error from getClusterVpc function, which can be different, for example - fail to get clusterId/getting vpcs/ vpc not found?

Reply:
ah, nvm! I think I just need to look into .Wrap stuff when I get a minute! It looks ok now! :)